hibp.js

```diff
@@ -101,7 +101,9 @@ const HIBP = {
   async getUnsafeBreachesForEmail(sha1, allBreaches) {
     const allFoundBreaches = await this.getBreachesForEmail(sha1, allBreaches, true);
     return allFoundBreaches.filter(
-      breach => !breach.IsSpamList
+      breach => !breach.IsSpamList &&
+                !breach.IsFabricated &&
```

Comment:
The logic here changed to now exclude any `.IsFabricated` breaches, is that expected?

Reply:
Expected, but potentially unnecessary. I think there is only one "breach.IsFabricated" and it gets filtered out by `!breach.IsVerified` (added back in [24450c4](https://github.com/mozilla/blurts-server/pull/663/commits/24450c4)).

Reply:
I think I lack a lot of context/knowledge to be the best reviewer... Perhaps we can summon @groovecoder to give a set of 👀 as well.

I think my confusion is around how we're loading and filtering breaches and why the `.IsSpamList` and `.IsFabricated`, and other flags logic lives in multiple places.

Can/should we do that filtering somewhere when we cron-import the breaches list in `loadBreachesIntoApp()`? If we know we'll never show spam list results, it may make sense to just filter them at import before shimming them into `app.locals.breaches` array. Then the only checks we may need to do are checking whether the breach has `.IsSensitive` set or not (depending on the context of web results vs emails). It may simplify the logic somewhat.
But like I say, maybe I'm missing something obvious and we're already doing that somewhere and I should simmer down.

https://github.com/mozilla/blurts-server/blob/e3cecb9a31065909ab20b995f0647bd4fd3395db/hibp.js#L81-L99

Reply:
Hey, sorry for the lack of context here. So just for background, last week we sent out an alert for the "You've Been Scraped" breach. Since "You've Been Scraped" isn't associated with a website, it caused a lot of confusion and resulted in a number of users replying to the alert wondering wth.

While content is off creating new copy to address the non-website breaches (spam lists included) of the future, we want to temporarily disable alerts for them and hide them everywhere except the "?breach=" pages. Once we have new content, these breach categories will be reintroduced but no word yet as to whether we'll be sending alerts for them, disclosing them in website scan results, or restricting them to email reports.

We shouldn't have been sending alerts for unverified or fabricated breaches to begin with, and they could be filtered out in loadBreachesIntoApp() except for the fact that we <i>are</i> still showing them on the "/?breach=" pages. I'm going to find out if this is still desired behavior, given that the doorhanger doesn't appear on sites with unverified or fabricated breaches. If it's not, we'll get rid of them and any other unshowables in loadBreachesIntoApp(), delete their logos, and forget all about them.